test/unit/specs/snapshotter.spec.js

```diff
@@ -0,0 +1,48 @@
+var Map = require('ti.map');
+const ANDROID = Ti.Platform.osname === 'android';
+if (!ANDROID) {
+	var snapshotter = Map.createSnapshotter({
+		mapType: Map.NORMAL_TYPE,
+		region: {
+			latitude: 49.261902,
+			longitude: -123.12494706,
+			latitudeDelta: 0.4,
+			longitudeDelta: 0.4,
+		},
+		size: { height: 200, width: 300 },
+		showsBuildings: true,
+		showsPointsOfInterest: false,
+	});
+
+	describe('ti.map.Snapshotter', function () {
+		it('should have deifined', () => {
+			expect(snapshotter).not.toEqual(undefined);
+		});
+
+		it('should have function setMapType', () => {
+			expect(snapshotter.setMapType).toEqual(jasmine.any(Function));
+		});
+
+		it('should have function setRegion', () => {
+			expect(snapshotter.setRegion).toEqual(jasmine.any(Function));
+		});
+
+		it('should have function setSize', () => {
+			expect(snapshotter.setSize).toEqual(jasmine.any(Function));
+		});
+
+		it('should  have function setShowsBuildings', () => {
+			expect(snapshotter.setShowsBuildings).toEqual(jasmine.any(Function));
+		});
+
+		it('should have function setShowsPointsOfInterest', () => {
+			expect(snapshotter.setShowsPointsOfInterest).toEqual(
+				jasmine.any(Function)
+			);
+		});
+
+		it('should have function takeSnapshot', () => {
+			expect(snapshotter.takeSnapshot).toEqual(jasmine.any(Function));
```

Comment:
Can you write a test case to see, if it is taking snapshot or not? One with invalid values if possible.

Reply:
Added UT please verify.